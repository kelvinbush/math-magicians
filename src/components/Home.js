import React from 'react';

const Home = () => <div />;

Home.propTypes = {};

export default Home;
