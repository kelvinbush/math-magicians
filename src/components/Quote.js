import React from 'react';

function Quote() {
  return (
    <div className="quote">
      <p>“The best way to predict the future is to create it.”</p>
      <p>- Abraham Lincoln</p>
    </div>
  );
}

export default Quote;
