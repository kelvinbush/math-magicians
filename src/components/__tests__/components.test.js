import renderer from 'react-test-renderer';
import Quote from '../Quote';
import Home from '../Home';

describe('Test Quote and Home component', () => {
  it('should render Quote component correctly', () => {
    const tree = renderer.create(<Quote />).toJSON();
    expect(tree).toMatchSnapshot();
  });

  it('should render Home Component correctly', () => {
    const tree = renderer.create(<Home />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
