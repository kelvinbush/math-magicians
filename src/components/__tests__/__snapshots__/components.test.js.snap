// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test Quote and Home component should render Home Component correctly 1`] = `
<div
  className="home"
>
  <h3>
    Welcome to our page!
  </h3>
  <p>
    lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum.
  </p>
  <p>
    Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum.
  </p>
</div>
`;

exports[`Test Quote and Home component should render Quote component correctly 1`] = `
<div
  className="quote"
>
  <p>
    “The best way to predict the future is to create it.”
  </p>
  <p>
    - Abraham Lincoln
  </p>
</div>
`;

exports[`renders correctly 1`] = `
<div
  className="quote"
>
  <p>
    “The best way to predict the future is to create it.”
  </p>
  <p>
    - Abraham Lincoln
  </p>
</div>
`;
